woodwork/column_accessor.py

```diff
@@ -85,13 +88,58 @@ def __eq__(self, other):
             return False
         return self._series.equals(other._series)
 
+    def __getattr__(self, attr):
+        '''
+            If the method is present on the Accessor, uses that method.
+            If the method is present on DataFrame, uses that method.
+        '''
+        if self._schema is None:
+            raise AttributeError("Woodwork not initialized for this Series. Initialize by calling Series.ww.init")
+        if hasattr(self._series, attr):
+            return self._make_series_call(attr)
+        else:
+            raise AttributeError(f"Woodwork has no attribute '{attr}'")
+
     def __repr__(self):
         msg = u"<Series: {} ".format(self._series.name)
         msg += u"(Physical Type = {}) ".format(self._series.dtype)
         msg += u"(Logical Type = {}) ".format(self.logical_type)
         msg += u"(Semantic Tags = {})>".format(self.semantic_tags)
         return msg
 
+    def _make_series_call(self, attr):
+        '''
+        Forwards the requested attribute onto the series object.
+        Intercepts return value, attempting to initialize Woodwork with the current schema
+        when a new Series is returned.
+        Confirms schema is still valid for the original Series.
+        '''
+        series_attr = getattr(self._series, attr)
+
+        if callable(series_attr):
+            def wrapper(*args, **kwargs):
+                # Make Series call and intercept the result
+                result = series_attr(*args, **kwargs)
+
+                # Try to initialize Woodwork with the existing Schema
+                if isinstance(result, pd.Series):
+                    if result.dtype == self._schema['logical_type'].pandas_dtype:
+                        # We don't need to pass dtype from the schema to init
+                        schema = {key: value for key, value in self._schema.items() if key != 'dtype'}
+                        result.ww.init(**schema)
+                    else:
+                        invalid_schema_message = f'Operation performed by {attr} has invalidated the Woodwork ' \
+                            'typing information:\n' \
+                            f'dtype mismatch between original dtype, {self._schema["logical_type"].pandas_dtype}, ' \
+                            f'and returned dtype, {result.dtype}.\n' \
+                            'Please initialize Woodwork with Series.ww.init'
+                        warnings.warn(invalid_schema_message, TypingInfoMismatchWarning)
+                # Always return the results of the Series operation whether or not Woodwork is initialized
+                return result
```

Comment:
Is the reason that we don't need to worry about the schema getting invalidated inplace because you can't change the dtype of a series inplace?

Reply:
Basically yes. A dtype change was the only scenario I thought of that would invalidate (since we don't have column changes to worry about here), and since you can't modify the dtype in place I think we should be good. We can do inplace modifications (like `series.pop()` for example), but I think that should be safe.

Reply:
Cool! 

Also not storing the name on the schema means that we can't invalidate it that way, which was the other way I was originally thinking could invalidate the schema!